src/app/app.component.ts

```diff
@@ -67,7 +71,16 @@ export class AppComponent {
     this.activatedRoute.params.subscribe(() => {
       this.loginService.login();
     });
-
+    // initial value
+    if (this.toggleAckService.showIconChanged) {
+      this.toggleAckService.getToggle().subscribe(val => {
+        this.show = val ? 'on' : 'off';
+      });
+      // subscribe for changes
+      this.toggleAckService.showIconChanged.subscribe(val => {
+        this.show = val.value ? 'on' : 'off';
```

Comment:
This surprised me. Why is "show" a string of `on|off` rather than simply a boolean?